sdks/go/pkg/beam/core/runtime/xlangx/expand.go

```diff
@@ -20,25 +20,40 @@ import (
 
 	"github.com/apache/beam/sdks/go/pkg/beam/internal/errors"
 	jobpb "github.com/apache/beam/sdks/go/pkg/beam/model/jobmanagement_v1"
+	"github.com/apache/beam/sdks/go/pkg/beam/model/pipeline_v1"
```

Comment:
Please abbreviate the pipeline_v1 package protos as pipepb.

Reply:
Done.